provider/provider.go

```diff
@@ -60,7 +61,7 @@ func (p *BaseProvider) getConfiguration(defaultTemplateFile string, funcMap temp
 	var defaultFuncMap = template.FuncMap{
 		"replace":   replace,
 		"tolower":   strings.ToLower,
-		"normalize": normalize,
+		"Normalize": Normalize,
```

Comment:
I suppose that a regression might have been introduced here as no template have been modified. We should leave the first one in low caps to stay consistent `"normalize": Normalize`.